app/gig.py

```diff
@@ -8,7 +8,9 @@
 
 
 class GIG(object):
-    __dest = os.getcwd() + '/.gitignore'
+    __base = os.getcwd()
+    __dest = __base + '/.gitignore'
+    __git  = __base + '/.git'
```

Comment:
Lets just use single underscores, shall we? :smile:

Reply:
I've used double underscores to mark them as private variables. I thought it was necessary to protect them from external modifications.